testsuite/MDAnalysisTests/analysis/test_hydrogenbonds_analysis.py

```diff
@@ -87,18 +87,111 @@ def test_count_by_type(self, h):
         counts = h.count_by_type()
         assert int(counts[0, 2]) == ref_count
 
-    def test_count_by_ids(self, h):
+    def test_count_by_ids(self, h, universe):
 
         ref_counts = [1.0, 1.0, 0.5, 0.4, 0.2, 0.1]
         unique_hbonds = h.count_by_ids()
 
+        most_common_hbond_ids = [12, 14, 9]
+        assert np.allclose(unique_hbonds[0,:3], most_common_hbond_ids)
```

Comment:
We prefer numpy's array comparison functions over bare asserts
```suggestion
        assert_equal(unique_hbonds[0,:3], most_common_hbond_ids)
```
See https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide%3A-Tests